src/frontend-scripts/components/section-main/Changelog.jsx

```diff
@@ -107,6 +107,21 @@ class Changelog extends React.Component {
 					<li> Mell0: 1907 </li>
 					<li> RyanLockwood: 1905 </li>
 				</ol>
+				

```

Comment:
I'd probably also toss in a `Welcome to season 10!` and the date `April 1, 2020` just since we didn't have an update to add the date there